### PR TITLE
Add an option to make issue submitter email private

### LINKED ISSUE #91: Additional options for user email so it's not displayed in a public repository
I want to encourage users to leave their email when submitting an issue so that we can follow up with them, but I don't think they will be comfortable with the fact that their email will then be listed with the issue in a public repository.  I am not familiar enough with GitHub issues to know what other options would be, but it would be nice to have some.

### PR DESCRIPTION
**Pull request by mafzst**
As suggestied by #91, add a checkout to let subitter choosing to send in the issue body his email or not
Close #91

### CODE REVIEW

**Comment by schneidmaster** [comment]
Thanks for the PR! Would you mind adding this to the `config/locales/pl.yml` (mirroring the English locale text you added): `E-mail Publiczne (wysyłanie w organizmie emisyjnej)`

It's a rough translation but since we have a Polish translation (submitted by a separate PR a while back) I'd like to keep it up to date.

**Comment by mafzst** [comment]
Yes of course :)

File: app/services/github_service.rb [hunk 1]
```diff
@@ -28,8 +28,8 @@ def load_repositories(access_token)
       # Add repos and remove user from any repos they no longer have access to
       remove_outdated(user, outdated_repos)
     end
-
-    def submit_issue(repo_id, sub_name, email, title, details)
+    
+    def submit_issue(repo_id, sub_name, email, email_public, title, details)
```
Comment (schneidmaster) [inline app/services/github_service.rb]: You should just need to add `# rubocop:disable Metrics/ParameterLists` before the definition, `# rubocop:enable Metrics/ParameterLists` after, and then also remove the trailing whitespace on line 31 to satisfy Rubocop.

Comment (mafzst) [inline app/services/github_service.rb]: Seems to be unsuccessful at https://github.com/schneidmaster/gitreports.com/pull/98/commits/1710441b4e1c21bc6e9fd462d2cd786b32a1ad2d

Comment (mafzst) [inline app/services/github_service.rb]: @schneidmaster I couldn't find something to disable this lint. Have you something new about that?

**Pull request merged**